index.js

```diff
@@ -2572,6 +2574,25 @@
   }
   S.not = def('not', {}, [$.Boolean, $.Boolean], not);
 
+  //# complement :: (a -> Boolean) -> a -> Boolean
+  //.
+  //. Takes a unary predicate and a value of any type, and returns logical
+  //. negation of applying predicate to the value.
```

Comment:
I'd like to see "the" used in two more places:

> Takes a unary predicate and a value of any type, and returns <ins>the</ins> logical negation of applying <ins>the</ins> predicate to the value.

Other than that, this description is excellent.